Library/Homebrew/keg.rb

```diff
@@ -292,6 +292,23 @@ def version
     PkgVersion.parse(path.basename.to_s)
   end
 
+  def to_formula
+    Formulary.from_keg(self)
+  end
+
+  def installed_dependents
```

Comment:
Could make it a static method or pull out into another class entirely so there's less going on in `uninstall`.